opat/Moodle-3.yaml

```diff
@@ -0,0 +1,807 @@
+title: Moodle Accessibility Conformance Report
+product:
+  name: Moodle
+  version: 3.1
+  description: The world's most customisable and trusted open-source learning management system.
+author:
+  name: Sander Bangma & Jun Pataleta
+  company_name: Moodle Pty Ltd
+  address: PO Box 303, West Perth WA 6872, Australia
+  email: support@moodle.com
+  website: https://docs.moodle.org
+notes:
```

Comment:
The validator complains when this is provided with no data. If you do `notes: ""` that is valid or alternatively remove the reference altogether.

Reply:
@mgifford I think all the current validation issues are the same, empty notes. So the fix above will work for all of them.